server/src/file_operations.ts

```diff
@@ -40,7 +40,8 @@ class DefaultProvider implements FsProvider {
   }
 
   public async glob(pattern: string) {
-    const found = sync(pattern, {nosort: true, nodir: true});
+    const options = {nosort: true, nodir: true};
```

Comment:
Did this because typescript complains about nosort not existing
Could have simply removed it, but in case it does exist but there's a typing error...